### DISCUSSION TITLE
Any recommended tutorial for building a world similar to the new SCARA T6?

### DISCUSSION THREAD

**Discussion by thephet** [cyberbotics/webots#5241] [Q&A]
Hi,

First of all thanks for this great piece of software. I have been using it for a while for doing very basic things. I wonder if there's any tutorial to create more complex environments like the new one with the SCARA T6. I mean a tutorial where you have a belt with objects and the robot picks and places them.

I have done all the tutorials in the documentation, but I don't know how to do something like this.

Thanks for your help!

**Comment by omichel** [comment]
There is no such advanced tutorial, but I would recommend you to start from an existing example which is close enough to your goal and modify it step-by-step to achieve exactly what you want. We have several examples with a conveyor belt and a robot picking and placing objects, like this one: https://cyberbotics.com/doc/guide/ure (see video). It should be easy to delete objects and add new ones from the user interface. Note: the simulation should be stopped at time 0 to make modifications and save them.
